### DISCUSSION TITLE
livewire + stored procedure

### DISCUSSION THREAD

**Discussion by sixtyrc** [livewire/livewire#4516] [1. Help]
Hello, I would need if someone knows an example of how to use livewire but without using a model, I need to use a stored procedure in an ms sql, but I can't find examples all of them always use a model, thanks.-

**Comment by joshhanley** [comment]
@sixtyrc not sure what you mean without using a model? Livewire can be used with primitive properties (string, array, etc) is that what you mean?

**Comment by sixtyrc** [comment]
Hello, if what I need to do is use datatables with livewire, but I use
stored procedures in SQL, but I don't get a clear example of how to
show my query returned by the procedure in the livewire datables

**Reply by joshhanley** [reply]
@sixtyrc well that depends on what format your stored procedure returns data. Do you have an example of an output from your stored procedure?

Are you talking about the Livewire datatables package? If so, that is a third party package, so you might be better off raising an issue on that packages repo instead.

**Comment by sixtyrc** [comment]
Hi,  I would like you to show them in a grid using the benefits of livewire

I call my procedure like this, this is part of my controller

$departures = DB::select('EXEC IHPatientsDepartedByDate "' . $month . '","'
. $year . '"');
          return view('outgoing', ['outgoing' => $outgoing]);

Thank you

**Comment by joshhanley** [comment]
@sixtyrc from my comment above, can you provide this info?

> well that depends on what format your stored procedure returns data. Do you have an example of an output from your stored procedure?

**Comment by sixtyrc** [comment]
Sure, here it is.

[image: image.png]

[image: image.png]
thanks

**Reply by joshhanley** [reply]
@sixtyrc looks like that didn't come through

![image](https://user-images.githubusercontent.com/882837/151247240-56639876-ec66-4333-a339-ab8089f5f61e.png)
